Tasks/QuickPerfTest/task.json

```diff
@@ -97,6 +103,9 @@
                 "Brazil South": "Brazil South (Sao Paulo State)",
                 "Australia East": "Australia East (New South Wales)",
                 "Australia Southeast": "Australia Southeast (Victoria)"
+            },
+            "properties": {
+                "EditableOptions": "True"
```

Comment:
Any reason to make this editable ? (We don't support all the azure data centers as we need capacity allocations done)

Reply:
I have no doubt about the good reason for limiting the choices and they work perfectly for VSTS. The TFS scenario requires more flexibility as upgrades happens way less frequently, which means that the Tasks get updated, if you are lucky, maybe two or three times a year if not every two to three years. In the meantime the Azure landscape change and you offer the option of an additional Region: TFS customers will be stuck with a old limited list.
The help should link to a doc page that clarifies and list the supported Regions.